sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java

```diff
@@ -30,6 +30,7 @@
 import com.sonar.orchestrator.container.SonarDistribution;
 import com.sonar.orchestrator.db.Database;
 import com.sonar.orchestrator.db.DefaultDatabase;
+import com.sonar.orchestrator.http.HttpCall;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.sonar.orchestrator.http.HttpCall'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)